tests/testthat/test_flex_table1.R

```diff
@@ -55,3 +55,9 @@ test_that("flex_table1 warns or stops when formula is not specified correctly",{
                              data = biv_data,
   ))
 })
+
+# Testing PR
```

Comment:
I think I found something, but it would require you to experiment with git in the terminal. 
Would you be willing to try this out? https://stackoverflow.com/a/51400593/7318488
It involves the interactive rebase, where you can decided for every commit if it should be included or not.

If you dont just do a "revert" commit (i.e., deleting the change and commit / push again).

Reply:
I usually avoid reverting commits because they're damn complicated 😅 so usually I just make the changes again in a new commit!

Reply:
That's fine as well :)